dnf/persistor.py

```diff
@@ -28,40 +28,38 @@
 from dnf.i18n import _
 import distutils.version
 import dnf.util
+
+import copy
```

Comment:
If all `copy.deepcopy(default)` calls will be removed then we do not need to import it.